integration/spark/src/main/common/java/io/openlineage/spark/agent/lifecycle/plan/KafkaRelationVisitor.java

```diff
@@ -130,7 +130,7 @@ public boolean isDefinedAt(LogicalPlan x) {
                                     fieldCount,
                                     Spliterator.SIZED & Spliterator.IMMUTABLE),
                                 false);
-                          } catch (JsonProcessingException e) {
+                          } catch (IOException e) {
```

Comment:
Was `JsonProcessingException` not the correct exception to handle?

Reply:
`ObjectMapper.readTree` is defined as 
```java
JsonNode readTree(String content)
        throws IOException, JsonProcessingException
```
The Jackson version we were compiling against wasn't the same as what is used in Spark, so when I fixed that it caused a compilation failure, as we weren't handling the `IOException`.